site/en/docs/extensions/reference/scripting/index.md

```diff
@@ -2,26 +2,24 @@
 api: scripting
 ---
 
-## Manifest
+## Manifest {: #manifest }
 
-In order to use the `chrome.scripting` API, you need to specify a
-`"manifest_version"` of `3` or higher and include the `"scripting"` permission
-in your [manifest file][manifest].
+To use the Scripting API, you need to declare the `"scripting"` permission in the [manifest][manifest] plus the host permissions for the pages you want to inject scripts into. You can either use the [`"host_permissions"`][match-patterns] key or the [activeTab][activetab] permission, which grants temporary host permissions. The following example uses the activeTab permission.
```

Comment:
Similar to the discussion in https://github.com/GoogleChrome/developer.chrome.com/pull/5050, I'd prefer to use `chrome.scripting` instead of the "Scripting API". Do you have any objections to changing that back here and below?